src/views/applicationManager.js

```diff
@@ -0,0 +1,24 @@
+const interview = {};
+interview.navigate = function(page) {
+    const appEl = document.querySelector('#app');
+    switch (page) {
+    case 'evaluations':
+        appEl.innerHTML = interview.EvaluationsPage();
```

Comment:
So far, so good. Now, let's move all of the module-specific rendering and event setup into a single setup method per module.

Reply:
yes, that's the next step I'm going to do :)